packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -800,6 +806,696 @@ let rows = [
 </Table>
 ```
 
+
+## Resizable Columns
+
+For resizable column support, two additional hooks need to be added to the table implementation above. The <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} />
+hook from `@react-stately/table` is responsible for initializing and tracking the widths of every column in your table, returning functions that you can use to
+update the column widths during a column resize operation. Note that this state is supplementary to the state returned by <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableState} />.
+
+The second column resizing hook is <TypeLink links={docs.links} type={docs.exports.useTableColumnResize} />. This hook handles the interactions for a table column's resizer
+element, allowing the user to drag the resizer or use the keyboard arrows to expand the column's width. Be sure to pass the state returned by <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} />
+to this hook so the tracked widths can be updated appropriately. We'll walk through all the required changes to the previous table implementation step by step below. For simplicity's sake, we'll be
+omitting support for selection and sorting.
+
+### Table
+
+As mentioned previously, we first need to call <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} /> to initialize the widths for our table's columns.
+We'll pass the state returned by <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} /> along with any user defined `onResize` handlers
+to our `ResizableTableColumnHeaders` so it can be used by <TypeLink links={docs.links} type={docs.exports.useTableColumnResize} />. The `widths` from this state are provided to each table element so
+the table cells in the body match their parent column's widths at all times.
+
+The various style changes below are to make the table itself scrollable when the row content overflows and to support table body/column widths greater than the 300px applied to the table itself.
+
+<details>
```

Comment:
Flipped back to collapsing the whole code sections blocks, better experience on mobile IMO

Reply:
I would expand them. The code is sorta the point of the docs I think. Most people will be looking at this on desktop as well.